app/src/js/actions/index.js

```diff
@@ -739,6 +739,15 @@ export const listRules = (options) => (dispatch, getState) => {
   });
 };
```

Comment:
Probably we only need to remove timeFilters from listRules.
https://github.com/nasa/cumulus-dashboard/pull/1208/files#diff-b437e4d6db394a61a39ac02fafdf1ffee8356fe7172db3a0b7e21c87815da4b0R731